notebooks/text_classification/solutions/word2vec.ipynb

```diff
@@ -0,0 +1,1453 @@
+{
```

Comment:
<p>Why not replacing the two cells above with instructions to run the <code>tensorbaord</code>  commend in cloudshel. It seems simpler that spinning an instance only for that purpose. </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/154/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='106'/>

Reply:
<p>I followed your suggestion and used tensorboard in gcloud shell</p>